### PR TITLE
add data for BW

### PR DESCRIPTION
**Pull request by mehlukas**
Data source: [gruene-bw.de/partei/kreisverbaende](https://www.gruene-bw.de/partei/kreisverbaende/)
I got the data from the html of the link above (and /page/[2](https://www.gruene-bw.de/partei/kreisverbaende/page/2/)-[5](https://www.gruene-bw.de/partei/kreisverbaende/page/5/)) and extracted it with regexing and a python script that uses the xml.etree library.

Currently, some lines in the yaml file seem to be too long.

Please give feedback and review.

### CODE REVIEW

**Comment by mehlukas** [comment]
Todo:
- [ ] Handle lines that are too long - How to break lines in YAML? [see here](https://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines#21699210)
- [ ] Consolidate facebook links
- [ ] Links should be unquoted

**Comment by mehlukas** [comment]
@marians Should we introduce some kind of line breaking or should we enlarge the limit of yamllint? If readability and easy usage is our main goal then I'd prefer enlarging the limit. (See [link](https://yamllint.readthedocs.io/en/stable/configuration.html#extending-the-default-configuration) how to do that)

**Comment by marians** [comment]
Since line breaking may be a bit difficult to get right in automation, I suggest to relax the yamllint rules, e. g. 

    yamllint -d "{extends: default, rules: {line-length: {max: 120}}}" <path>

File: data/countries/de/bw/data.yaml [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+---
+country: DE
+district: Aalen/Ellwangen
+emails:
+  - address: kv.aalen-ellwangen@gruene.de
+level: DE:KREISVERBAND
+state: Baden-Württemberg
+type: REGIONAL_CHAPTER
+urls:
+  - type: WEBSEITE
```
Comment (marians) [inline data/countries/de/bw/data.yaml]: This should be `WEBSITE`. Same for other occurrences.

**Comment by mehlukas** [comment]
@marians what else needs to be changed for travis to check again? (I changed the .travis.yml file)

**Comment by marians** [comment]
Normally Travis should have picked up the last commit, but it hasn't. Now it's in a weird state ("Waiting for status to be reported").

You can try and push an empty commit to trigger another build, o simply add an empty line to README.md or something.

**Comment by mehlukas** [comment]
doesn't seem to work

**Comment by marians** [comment]
I suggest to wait. Maybe Travis is acting up.

**Comment by marians** [comment]
I guess I found the problem. It's in the `.travis.yml`

image

**Comment by marians** [comment]
@mehlukas I took the liberty to fix the travis config syntax. You can proceed.

**Pull request merged**